dashboard/dashboard.html

```diff
@@ -314,9 +313,17 @@ <h2>Amherst COVID Data Dashboard</h2>
         var active_cases = data['raw'][n - 1];
         var last_ma = data['ma'][n - 14 - 1];
         var delta = (data['ma'][n - 1] - last_ma) / last_ma;
+        var delta_direction;
+        if (delta > 0) {
+          delta_direction = "&#8593;"
```

Comment:
can we make it colorful? 0:-)
It's a bit more painful, but we can just make a span behind and change the style with the same JS code..